tests/kernels/test_attention.py

```diff
@@ -111,6 +111,7 @@ def ref_single_query_cached_kv_attention(
         output[i].copy_(out, non_blocking=True)
 
 
+@pytest.mark.skip(reason="speed")
```

Comment:
Could we leave a more clear trace that this is our change and choice? At least "NM slow".

Maybe you could make a slow marker and use that rather than always skipping this test https://docs.pytest.org/en/latest/how-to/mark.html